utils/wasm-info/Cargo.toml

```diff
@@ -4,6 +4,6 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-hex.workspace = true
+hex = { workspace = true, features = ["std"]}
```

Comment:
why "std" feature now required?

Reply:
Dependencies for `wasm-info` were changed to be managed via workspace. Before this change, `wasm-info` relied on the `hex` crate with default features, but now it does not, which caused the crate to fail to build. To fix this, I added the minimal required feature set to allow `wasm-info` to work again